staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/README.md

```diff
@@ -0,0 +1,4 @@
+This directory contains a port of https://github.com/google/cel-policy-templates-go/tree/master/policy/model modified in a few ways:
```

Comment:
include a copy of https://github.com/google/cel-policy-templates-go/blob/master/LICENSE at staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/